src/main/resources/log4j2.properties

```diff
@@ -3,7 +3,7 @@ appender=console
 appender.console.type = Console
 appender.console.name = STDOUT
 appender.console.layout.type = PatternLayout
-appender.console.layout.pattern = {"timestamp": "%d{YYYY-MM-DD HH:mm:ss.SSS}", "level" : "%-5p", "thread" : "%T", "logger" : "%c{1}:%L", "message" : "%msg", "context" : %msg%n 
+appender.console.layout.pattern = {"timestamp": "%d{YYYY-MM-dd HH:mm:ss.SSS}", "level" : "%-5p", "thread" : "%T", "logger" : "%c{1}:%L", "message" : "%msg", "context" : %msg%n 
```

Comment:
Pattern is missing the closing bracket "}". 
Property "%msg" is duplicated, replace value of context key with "default".